ui/app/adapters/pki/issuer.js

```diff
@@ -31,6 +33,45 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  async getIssuerMetaData(store, type, query, res) {
+    const { backend, shouldShowIssuerMetaData } = query;
+
+    const records = await allSettled(
+      res.data.keys.map((id) => {
+        return this.queryRecord(store, type, { id, backend });
```

Comment:
Is the only data we need from this query the certificate? If that's the case, and the rest already exists on the `LIST` endpoint I wonder if we can just fetch for the json certificate 🤔 Then we'd bypass the extra work that happens in the serializer for each issuer